packages/eslint-plugin/tests/rules/class-literal-property-style.test.ts

```diff
@@ -84,6 +84,39 @@ abstract class Mx {
         \`;
       }
     `,
+    `
+      class Mx {
+        set p1(val) {}
+        get p1() {
+          return '';
+        }
+      }
+    `,
+    `
+      let p1 = 'p1';
+      class Mx {
+        set [p1](val) {}
+        get [p1]() {
+          return '';
+        }
+      }
+    `,
```

Comment:
[Testing] Comments tend to trip up string checking, so it'd be good to have this (already working, yay!) test case:

```suggestion
    `,
    `
      let p1 = 'p1';
      class Mx {
        set [/* before set */ p1 /* after set */](val) {}
        get [/* before get */ p1 /* after get */]() {
          return '';
        }
      }
    `,
```